tools/Vrops.py

```diff
@@ -142,23 +139,23 @@ def get_resources(self, target: str, token: str, uuids: list, adapterkind: str,
             return resources, response.status_code
 
     def get_datacenter(self, target, token, parent_uuids):
-        return self.get_resources(target, token, parent_uuids, adapterkind="VMWARE", resourcekinds=["Datacenter"],
-                                  resource_obj=Datacenter)
+        return self.get_resources(target, token, parent_uuids, adapterkind="VMWARE",
+                                  resourcekinds=["Datacenter"], resource_class=Datacenter)
```

Comment:
This is always 1:1. How about we put `resourcekind` in respective `resource_class` and access it this way. I would make it optional, to either put `resource_class` or a `resourcekinds` list in this function. this way we could still request more than one, if needed.

Reply:
Yes, there is still room for improvement. I'll take that along for the next iteration. The idea is to make it configurable from the outside.

Reply:
Ok, to be tracked in #167